app/views/govuk_publishing_components/components/docs/select.yml

```diff
@@ -31,25 +31,52 @@ examples:
         selected: true
       - text: 'Option three'
         value: 'option3'
-  with_data_attributes:
-    description: Data attributes can be added to the option elements of the select. Note that the component will not do anything with them - Javascript will have to be added in the application that uses the component, for example to fire tracking events.
+  with_tracking:
+    description: 'Tracking can be enabled on the select component by passing a minimum of data_track_category and data_track_action. Other tracking attributes are optional. Note: tracking events do not currently fire within the component guide'
```

Comment:
Nitpick: I don't think the single quotes are needed? Also no full stop.

Reply:
The single quotes are needed for the "Note:" otherwise the YML can't parse it because of the colon. I'll add the full stop though 👍

Reply:
Ah, of course. Sorry, hadn't noticed that.